src/app/Application/Middleware/IsSuccessMiddleware.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GarageGroup.Internal.Timesheet;
+
+internal static partial class IsSuccessMiddleware
+{
+    private const string IsSuccessField = "isSuccess";
+
+    private const string IsSuccessFalseJson = "{\"isSuccess\":false}";
+
+    private const string IsSuccessTrueJson = "{\"isSuccess\":true}";
+
+    private const string ContentType = "application/json";
+
+    private const int SuccessStatusCode = 200;
+
+    private const int NoContentStatusCode = 204;
+
+    private const string ProblemDetailsSchemaName = "ProblemDetails";
+
+    private static readonly OpenApiSchema OpenApiSuccessSchema
+        =
+        new()
+        {
+            Type = "boolean",
+            Example = new OpenApiBoolean(true)
+        };
+
+    private static readonly OpenApiSchema OpenApiFailureSchema
+        =
+        new()
+        {
+            Type = "boolean",
+            Example = new OpenApiBoolean(false)
+        };
+
+    private static readonly JsonSerializerOptions SerializerOptions
+        =
+        new()
+        {
+            WriteIndented = true
+        };
+
+    private static void InnerConfigureSwagger(OpenApiDocument openApiDocument)
+    {
+        var paths = openApiDocument.Paths.Values;
+
+        foreach (var path in paths)
+        {
+            var operations = path.Operations.Values.FirstOrDefault();
```

Comment:
Could you please tell me why you only process the first operation and ignore the rest?